site/styles/_footer.scss

```diff
@@ -108,12 +108,19 @@ footer {
   a {
     padding: 3px 0 6px 0;
 		margin: 0;
-		
 		.footer-festivalscope-logo {
-			filter: invert($footer-fs-invert-percentage);
+			@if lightness($body-color) > lightness($body-bg) {
+        filter: invert(0%);
+      } @else {
+        filter: invert(100%);
+      }
 		}
 		&:hover .footer-festivalscope-logo  {
-			filter: invert($footer-fs-invert-percentage-hover);
+			@if lightness($body-color) > lightness($body-bg) {
+        filter: invert(100%);
+      } @else {
+        filter: invert(0%);
+      }
```

Comment:
Can you construct the `@if` (badass btw!) so we don't need the `@else` and not have an extraneous `invert(0%)`?

Reply:
Also the whitespace looks mixed.

Reply:
Also the whitespace looks mixed.

Reply:
removed extraneous `invert(0%)` and fixed the whitespace issue